src/ws/Makefile-ws.am

```diff
@@ -166,7 +166,7 @@ testserviceddir = $(systemdunitdir)/cockpit.service.d
 
 install-tests::
 	mkdir -p $(DESTDIR)$(pamdir) $(DESTDIR)$(testserviceddir) $(DESTDIR)/etc/cockpit
-	$(INSTALL_DATA) mock-pam-conv-mod.so $(DESTDIR)$(pamdir)/
+	install mock-pam-conv-mod.so $(DESTDIR)$(pamdir)/
```

Comment:
`${INSTALL_PROGRAM}` or at least `${INSTALL}`, no?

not that I think it matters....

Reply:
Thanks! I was looking for something like that, but didn't find it. Updated to INSTALL_PROGRAM, works fine.